botCommands/points.test.js

```diff
@@ -1,79 +1,65 @@
-const commands = require('./points')
-const {generateLeaderData} = require('./mockData')
-const axios = require('axios')
-const {Guild, Channel, Client, User} = require('discord.js')
-axios.post = jest.fn()
-
-const mockSend = jest.fn()
-mockSend.mockImplementation(message => {
- return message
-})
-
-jest.mock('discord.js', () => {
-  return {
-    Client : jest.fn().mockImplementation((users, channel, user) => {
-      return {
-        channels : {
-          get : () => channel
-        },
-        users : {
-          get : (userId) => users.filter(user => `<@${userId}>` === user.id)[0]
-        },
-        user : user
-      }
-    }),
-
-    Guild : jest.fn().mockImplementation((users) => {
-      return {
-        members : {
-          members : users,
-          get : (id) => {
-            return users.filter(member => member.discord_id === id)[0]
-          }
-        },
-        roles : [{name : "club-40"}],
-        member : (user) => {
-         return users.filter(member => member === user)[0]
-        }
-      }
-    }),
-
-    Channel : jest.fn().mockImplementation(() => {
-      return {
-        send: mockSend
-      }
-    }),
-
-    User : jest.fn().mockImplementation((roles, id, points) => {
-      return {
-        roles: roles,
-        id: `<@${id}>`,
-        points: points,
-        addRole: () => {
-          roles.push('club-40')
-        },
-        toString : () => `<@${id}>`
-      }
-    })
-  }
-})
+const axios = require('axios');
+const {
+  Guild, Channel, Client, User,
+} = require('discord.js');
```

Comment:
I think this looks a bit off, this was from ESLint?

Reply:
Yes, I didn't touch that part of the code after running eslint autofix. Also disabled eslint and prettier before changing the example string above.